python/cudf/cudf/_lib/aggregation.pyx

```diff
@@ -354,6 +355,18 @@ cdef class Aggregation:
             ))
         return agg
 
+    @classmethod
+    def cov(cls,
+            libcudf_types.size_type min_periods,
+            libcudf_types.size_type ddof=1
+            ):
```

Comment:
Indent style should be the following (code formatters like `black` don't support Cython):
```suggestion
    def cov(
        cls,
        libcudf_types.size_type min_periods,
        libcudf_types.size_type ddof=1
    ):
```